tests/model_forms/test_modelchoicefield.py

```diff
@@ -85,6 +85,17 @@ def test_model_choice_null_characters(self):
         with self.assertRaisesMessage(ValidationError, msg):
             f.clean("\x00something")
 
+    def test_model_choice_invalid_pk_value_error_messages(self):
+        uuid_f = forms.ModelChoiceField(UUIDPK.objects.all())
+        f = forms.ModelChoiceField(Category.objects.all())
+        for model_choice_form in [f, uuid_f]:
```

Comment:
In general, I'd try to avoid loops like this in tests because it makes debugging a failure (which iteration failed). There is `subTest`, however, as far as I can tell, we only need to test the UUID case for this fix. Adding additional test coverage (which I'm not sure is needed in this case) should always be done in a separate commit.

Reply:
I agree as well. I will incorporate the feedback and submit it again.

Reply:
I have applied the changes.

Reply:
Looks good. I only wonder if both tests should be in test_uuid.py.

Reply:
I believe that `ValidationError` is an issue that can occur with fields like `UUIDField`, so it seems more appropriate to place it in `test_uuid.py`.